css/templates/e-commerce/page.css

```diff
@@ -5,51 +5,265 @@
 @import 'page-vars';
 
 body {
-  font-size: var(--narrow-body-font-size);
-}
-
-@media (--breakpoint-md) {
-  body {
-    font-size: var(--wide-body-font-size);
-  }
+  font-size: var(--body-font-size);
+  line-height: initial;
 }
 
 /* TYPOGRAPHY */
 .h3 {
   color: var(--color-primary);
+}
+
+.h7 {
+  font-size: var(--h7);
+}
+
+h1, h2, h3, h4, h5, h6, h7,
+.h1, .h2, .h3, .h4, .h5, .h6, .h7 {
   line-height: normal;
-  padding-bottom: var(--space-1);
+  letter-spacing: 0;
+  font-family: var(--title-font);
+  text-transform: uppercase;
+  font-weight: bold;
+  color: var(--color-black);
+}
+
+.mb5 {
+  margin-bottom: var(--space-6);
+}
+
+.pb5 {
+  padding-bottom: var(--space-6);
+}
+
+@media (--breakpoint-md) {
```

Comment:
Most headers in the design change font size from narrow to wide. There isn't an obvious correlation so setting a new font size in the body wouldn't work. Some headers stay the same size across breakpoints whilst others do not.